package.json

```diff
@@ -8,7 +8,9 @@
     "install": "npm install --prefix packages/openactive-broker-microservice && npm install --prefix packages/openactive-integration-tests",
     "prettier": "prettier --check \"packages/**/*.js\" \"packages/**/*.json\"",
     "prettier:write": "prettier --write \"packages/**/*.js\" \"packages/**/*.json\"",
-    "start": "npm start --prefix packages/openactive-broker-microservice",
+    "start": "echo 'Error: please select a package to run and then run it e.g. `npm start --prefix packages/openactive-broker-microservice`' && exit 1",
+    "start-broker": "npm start --prefix packages/openactive-broker-microservice",
+    "start-tests": "npm start --prefix packages/openactive-integration-tests",
     "test": "npm test --prefix packages/openactive-integration-tests",
```

Comment:
Similarly `npm test` could run `npm test --prefix tests` ?

Reply:
@nickevansuk I'm not quite sure what you mean here. `npm test --prefix tests` would not run anything

Reply:
Updated based on other comment... can we have it run both package tests from `npm test` here?

Reply:
Yup. That's now done 👍